### PR TITLE
feat: added modal component (P1)

### PR DESCRIPTION
**Pull request by shraddha15-creator**
Added Modal component

- Live link to open modal added.

modal

### CODE REVIEW

File: pages/components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+.modal-content {
+    background-color: var(--primary-light);
+    text-align: center;
+    margin: auto;
+    padding: 20px;
+    width: 35%;
+    box-shadow: 3px 3px 15px var(--modal-bg-color);
+}
+
+#open-modal {
```
Comment (ScoobyDubeyDo) [inline pages/components/modal/modal.css]: Why you are grabbing the element by its id, use classes in place in place of this, as they can be used in multiple elements.

**Review by ScoobyDubeyDo — CHANGES_REQUESTED** [review]

**Comment by mohammadkhizerkhan** [comment]
use relative units especially in font-size.

File: pages/components/modal/modal.css (hunk 1 above)
Comment (shraddha15-creator) [reply pages/components/modal/modal.css]: Thanks for pointing out. Changed it to class now. please chack.

**Review by mohammadkhizerkhan — APPROVED** [review]
go for it

**Review by Pratik1005 — APPROVED** [review]

**Pull request merged**